include/stumpless/target.h

```diff
@@ -948,6 +949,49 @@ vstumpless_add_message( struct stumpless_target *target,
                         const char *message,
                         va_list subs );
 
+/**
+ *	Gets the current console stream where logs are written to. 	
+ *
+ * **Thread Safety: MT-Safe**
+ * This function is thread safe. Atomic variables are used to store and
+ * retrieve the error stream.
+ *
+ * **Async Signal Safety: AS-Safe**
+ * This function is safe to call from signal handlers.
+ *
+ * **Async Cancel Safety: AC-Safe**
+ * This function is safe to call from threads that may be ansynchronously
+ * cancelled.
+ *
+ * @return The current stream where messages are logged to on setting the
+ *			the CONS option.
+ */
+FILE *
+stumpless_get_cons_stream( void );
+
+/**
+ * Sets the console stream to write logs to.
+ *
+ * This will be stdout by default, but can be set to any stream. If it is set
+ * to NULL then error messages will not be printed.
+ *
+ * **Thread Safety: MT-Safe**
+ * This function is thread safe. Atomic variables are used to store and
+ * retrieve the error stream.
+ *
+ * **Async Signal Safety: AS-Safe**
+ * This function is safe to call from signal handlers.
+ *
+ * **Async Cancel Safety: AC-Safe**
+ * This function is safe to call from threads that may be ansynchronously
+ * cancelled.
+ *
+ * @param stream The stream to write logs to. If this is NULL then it will be
+ * ignored
```

Comment:
The last sentence is unfortunately misleading in the error stream function. For this new one, change it to something more explicit such as "If this is NULL then messages intended for the console will be ignored." or something similar.

Feel free to change the corresponding language in the error stream function as well!